libraries/cmake/source/augeas/config/macos/config.h

```diff
@@ -0,0 +1,1827 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* Define if building universal (internal helper macro) */
+/* #undef AC_APPLE_UNIVERSAL_BUILD */
+
+/* Define to the number of bits in type 'ptrdiff_t'. */
+/* #undef BITSIZEOF_PTRDIFF_T */
+
+/* Define to the number of bits in type 'sig_atomic_t'. */
+/* #undef BITSIZEOF_SIG_ATOMIC_T */
+
+/* Define to the number of bits in type 'size_t'. */
+/* #undef BITSIZEOF_SIZE_T */
+
+/* Define to the number of bits in type 'wchar_t'. */
+/* #undef BITSIZEOF_WCHAR_T */
+
+/* Define to the number of bits in type 'wint_t'. */
+/* #undef BITSIZEOF_WINT_T */
+
+/* Define to one of `_getb67', `GETB67', `getb67' for Cray-2 and Cray-YMP
+   systems. This function is required for `alloca.c' support on those systems.
+   */
+/* #undef CRAY_STACKSEG_END */
+
+/* Define to 1 if using `alloca.c'. */
+/* #undef C_ALLOCA */
+
+/* Define to 1 if the C locale may have encoding errors. */
+/* #undef C_LOCALE_MAYBE_EILSEQ */
+
+/* Define as the bit index in the word where to find bit 0 of the exponent of
+   'double'. */
+#define DBL_EXPBIT0_BIT 20
+
+/* Define as the word index where to find the exponent of 'double'. */
+#define DBL_EXPBIT0_WORD 1
+
+/* Define to 1 if // is a file system root distinct from /. */
+/* #undef DOUBLE_SLASH_IS_DISTINCT_ROOT */
+
+/* whether debugging is enabled */
+#define ENABLE_DEBUG 1
```

Comment:
Interesting, I wonder what effect this has vs. `0`?

Reply:
That seems to be on on the Linux side too, but anyway it enables some debug helpers:
https://github.com/hercules-team/augeas/blob/3775c2bf53fef5f694fcf25308cee1dfe00600c4/src/internal.c#L508

and https://github.com/hercules-team/augeas/blob/3775c2bf53fef5f694fcf25308cee1dfe00600c4/src/lens.c#L2365

Seems to compile fine with it off, you want me to remove it?

Reply:
Na this is OK, thanks for looking into it!

Reply:
If we do remove it, let's do it in a dedicated follow-up PR.